src/test/java/io/confluent/idesidecar/restapi/application/SidecarInfoTest.java

```diff
@@ -193,6 +249,7 @@ Stream<DynamicTest> testCombinations() {
               SidecarInfo sidecar = createSidecarInfo(input);
 
               // Verify the output matches
+              assert(input.userAgent).contains("support@confluent.io) sidecar/");
```

Comment:
Hmm, it seems we're not calling the newly added `userAgent()` method here and only asserting against the static value as provided by the test inputs. In order to properly test the method, I think this should instead be:

```suggestion
              assertEquals(input.userAgent, sidecar.userAgent())
```